### ISSUE TITLE
Calculate Indoor Air Quality

### ISSUE THREAD

**Issue by MarconiLab** [kriswiner/BME680#4]
Hi can you calculate from the gas resistance value o justa get the IAQ and Accuracy status? Thanks

**Comment by kriswiner** [comment]
Yes, I can measure resistance value. But I don;t use this sensor anymore, I
use the CCS811.
